mathesar/tests/integration/test_columns.py

```diff
@@ -8,3 +8,26 @@ def test_add_column(page, go_to_patents_data_table):
     page.click("button:has-text('Add')")
     column_header = f".table-content .header .cell .name:has-text('{column_name}')"
     expect(page.locator(column_header)).to_be_visible()
+
+
+def test_convert_text_col_of_num_to_num_col(page, patent_schema, base_schema_url):
+    page.goto(base_schema_url)
+    page.click("[aria-label='New Table']")
+    page.click("button:has-text('Empty Table')")
+    page.click("[aria-label='New Column']")
+    page.fill(".new-column-dropdown input", 'foo')
+    page.click("button:has-text('Add')")
```

Comment:
Is there a reason why we creating a new table _in_ this test?

We can use an already existing table like the patents table and use existing fixtures like `go_to_patents_data_table`, or if we desire to use an empty table, we can create new fixtures for creating and opening empty tables. Eg., we can use existing `empty_nasa_table` fixture and write a new `go_to_nasa_data_table` fixture and make use of it in this test.

Reply:
> Is there a reason why we creating a new table in this test?

@pavish I tried to replicate as shown in the issue description GIF, #1128

> We can use an already existing table like the patents table and use existing fixtures like `go_to_patents_data_table`, or if we desire to use an empty table, we can create new fixtures for creating and opening empty tables. Eg., we can use existing `empty_nasa_table` fixture and write a new `go_to_nasa_data_table` fixture and make use of it in this test.

Sure, this seems better, I'll make the required changes.